pkg/loop/server.go

```diff
@@ -60,21 +61,19 @@ type Server struct {
 	DataSource      sqlutil.DataSource // optional
 	promServer      *PromServer
 	checker         *services.HealthChecker
+	LimitsFactory   limits.Factory
```

Comment:
LOOP Plugins can construct Limits from this factory, which is wired up to beholder metrics, and will have dynamic updates plugged in seamlessly later on.